util/network-devp2p/src/service.rs

```diff
@@ -18,14 +18,19 @@ use std::net::SocketAddr;
 use std::ops::RangeInclusive;
 use std::sync::Arc;
 
+//TODO: remove this
```

Comment:
> TODOs should be tickets

Do we have a ticket for that :P

Reply:
I didn't add the `TODO` but it illustrates exactly my point: we're carrying around a todo-comment nobody even knows what it means anymore. The information content is tending towards zero...
I'll actually just remove it.